src/renderer/services/chain/fees/swap.ts

```diff
@@ -69,8 +83,14 @@ const feesByChain$ = (chain: Chain, memo?: Memo): FeesLD => {
 
 type SwapFeeType = 'source' | 'target'
 
-const swapFee$ = (asset: Asset, type: SwapFeeType, memo?: Memo): LiveData<Error, BaseAmount> => {
-  const feeByChain$: LiveData<Error, Fees> = feesByChain$(getChainAsset(asset.chain).chain, memo)
+const swapFee$ = (
+  asset: Asset,
+  type: SwapFeeType,
+  memo?: Memo,
+  amount?: BaseAmount,
+  recipient?: string
```

Comment:
[nitpick] Whenever we have more then two parameters, wrap parameters into a single object.

 ```ts
(
 asset: Asset,
  type: SwapFeeType,
  memo?: Memo,
  amount?: BaseAmount,
  recipient?: string
  )
  ```
:arrow_down: 

```ts
(
 {asset, type, memo, amount, recipient}: 
 {asset: Asset,
  type: SwapFeeType,
  memo?: Memo,
  amount?: BaseAmount,
  recipient?: string
  }
  )
  ```